examples/chem-sync-local-flask/local_app/benchling_app/views/canvas_initialize.py

```diff
@@ -25,6 +29,13 @@ def render_search_canvas(app: App, canvas_initialized: CanvasInitializeWebhookV2
         app.benchling.apps.create_canvas(canvas_builder.to_create())
 
 
+def render_search_canvas_for_created_canvas(app: App, canvas_id: str) -> None:
+    with app.create_session_context("Show Sync Search", timeout_seconds=20):
+        canvas_builder = _canvas_builder_from_canvas_id(app, canvas_id)
+        canvas_builder.blocks.append(input_blocks())
+        app.benchling.apps.update_canvas(canvas_id, canvas_builder.to_update())
```

Comment:
We can change this function to be more efficient since the [changes in SDK v1.20.0](https://docs.benchling.com/changelog/sdk-version-1200-release-notes) 

```py
def render_search_canvas_for_created_canvas(app: App, canvas_created: CanvasCreatedWebhookV2Beta) -> None:
    with app.create_session_context("Show Sync Search", timeout_seconds=20):
        canvas_builder = CanvasBuilder(app_id=app.id, feature_id=canvas_created.feature_id)
        canvas_builder.blocks.append(input_blocks())
        app.benchling.apps.update_canvas(canvas_id, canvas_builder.to_update())
```

This means we can entirely remove `_canvas_builder_from_canvas_id` (and also get some efficiency gains by not needing a GET call to the api via `get_canvas_by_id`)